Procfile

```diff
@@ -1,5 +1,5 @@
 # Run Rails & Webpack concurrently
 # If you wish to use a different server then the default, use e.g. `export RAILS_STARTUP='puma -w 3 -p 3000 --preload'`
-rails: [ -n "$RAILS_STARTUP" ] && $RAILS_STARTUP || bin/rails server -b 0.0.0.0
+rails: [ -n "$RAILS_STARTUP" ] && $RAILS_STARTUP || [ -n "$BIND" ] && $BIND bin/rails server -b $BIND || bin/rails server -b 127.0.0.1
```

Comment:
The `$BIND` before bin/rails appears superfluous? It causes a warning:

<pre>
13:11 $ BIND=0.0.0.0 foreman start                                                                                                                                                                                                                                              
13:11:08 rails.1   | started with pid 24145
13:11:08 webpack.1 | started with pid 24146
13:11:08 rails.1   | sh: 0.0.0.0: command not found
</pre>